tachyon/c/zk/plonk/halo2/BUILD.bazel

```
@@ -317,10 +337,16 @@ tachyon_cc_unittest(
     ],
     deps = [
         ":bn254_gwc_prover",
+        ":bn254_halo2_ls",
         ":bn254_shplonk_prover",
         "//tachyon/c/crypto/random:rng",
+        "//tachyon/c/math/polynomials:constants",
+        "//tachyon/c/math/polynomials/univariate:bn254_univariate_dense_polynomial",
+        "//tachyon/c/math/polynomials/univariate:bn254_univariate_evaluation_domain",
         "//tachyon/c/zk/plonk/halo2/test:bn254_halo2_params_data",
         "//tachyon/cc/math/elliptic_curves/bn/bn254:fr",
+        "//tachyon/math/elliptic_curves/bn/bn254",
         "//tachyon/math/finite_fields/test:finite_field_test",
+        "//tachyon/zk/lookup/halo2:lookup_scheme",
```

You can remove these dependencies since `bn254_halo2_ls` depends on them

```suggestion
        "//tachyon/c/zk/plonk/halo2/test:bn254_halo2_params_data",
        "//tachyon/cc/math/elliptic_curves/bn/bn254:fr",
        "//tachyon/math/finite_fields/test:finite_field_test",
```